static/bidder-info/melozen.yaml

```diff
@@ -0,0 +1,17 @@
+# We have the following regional endpoint domains: us-east and us-west
+# Please deploy this config in each of your datacenters with the appropriate regional subdomain
+endpoint: "https://prebid.melozen.com/rtb/v2/bid?publisher_id={{.PublisherID}}"
```

Comment:
endpoint is reachable
```
% curl -i --location --request POST 'https://prebid.melozen.com/rtb/v2/bid?publisher_id='
HTTP/1.1 204 No Content
Server: nginx/1.14.1
Date: Thu, 01 Aug 2024 05:56:03 GMT
Content-Type: application/json
Connection: keep-alive
```

Reply:
Thank you for confirming that the endpoint is reachable and behaves as expected. I appreciate your help in ensuring our configurations are correctly set up!